web/app/ui/MetalakeList.js

```diff
@@ -26,7 +26,7 @@ function TableToolbar(props) {
     <>
       <Fragment>
         <Portal container={() => document.getElementById('filter-panel')}>
-          <Box className={`twc-w-full twc-justify-between twc-hidden`}>
+          <Box className={` twc-flex twc-w-full twc-justify-between`}>
```

Comment:
What is the purpose of adding a white space before "twc-flex"?

Reply:
My mistake, I'll fix it.